config/database.yml

```diff
@@ -29,10 +29,10 @@ development:
   # To create additional roles in postgres see `$ createuser --help`.
   # When left blank, postgres will use the default role. This is
   # the same name as the operating system user that initialized the database.
-  username: postgres
+  # username: postgres
 
   # The password associated with the postgres role (username).
-  password: faith2019
+  # password: faith2019
 
```

Comment:
[Optional]
When you make changes to config/database.yml you can commit this file from your commits. That way others will not need to change it each time they pull updates.